lib/CFRMessageProvider.jsm

```diff
@@ -306,7 +306,6 @@ const CFR_MESSAGES = [
     id: "PIN_TAB",
     template: "cfr_doorhanger",
     category: "cfrFeatures",
-    exclude: true,
```

Comment:
Do we want to wait for the animation and bullet point formatting to actually turn enable this?